puf_data/StatMatch/Matching/runmatch.py

```diff
@@ -7,47 +7,39 @@
 from add_cps_vars import add_cps
 from add_nonfilers import add_nonfiler
 import pandas as pd
-import argparse
+import os
 
 """
     Script to run each phase of the matching process
 """
 
 
-def match(mar_cps_path='asec2016_pubuse_v3.dat',
-          puf_path='puf2011.csv'):
-    # Add arguments for specifying path to CPS file in CSV format
-    # this will allow the program to skip the process of creating the CPS from
-    # a .DAT file.
-    parser = argparse.ArgumentParser()
-    parser.add_argument('-c', '--cps', help='path to CPS file in CSV format')
-    parser.add_argument('-d', '--dat', help='path to CPS file in DAT format')
-    parser.add_argument('-p', '--puf', help='path to PUF file in CSV format')
-    args = parser.parse_args()
-
-    # Create CPS file either from a CPS or through create_cps method
-    if args.cps is not None:
-        mar_cps = pd.read_csv(args.cps)
+def match():
+    # If there is a .CSV version of the CPS, simply read that in. Otherwise
+    # convert the .DAT file to a .CSV
+    cps_csv_path = 'cpsmar2016.csv'
+    if os.path.isfile(cps_csv_path):
+        print('Reading CPS Data')
+        mar_cps = pd.read_csv('cpsmar2016.csv')
```

Comment:
Might want to have this line be:
```
mar_cps = pd.read_csv(cps_csv_path)
```
which is line the way you handled it below with the DAT file.